go.mod

```
@@ -1,12 +1,12 @@
 module github.com/bank-vaults/vault-sdk
 
-go 1.19
+go 1.22.2
```

i think pinning this should unblock the pipeline
```suggestion
go 1.22
```

---

Didn't work. :/